style.css

```diff
@@ -0,0 +1,376 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Quicksand:wght@300;400;500;600;700&family=Ubuntu+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap");
+
+/* ------------------- */
+/* Resets              */
+/* ------------------- */
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+p,
+figure,
+picture {
+  margin: 0;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  font-weight: 400;
+}
+
+img {
+  max-width: 100%;
+  display: block;
+}
+
+input,
+button,
+textarea,
+select {
+  font: inherit;
+}
+
+/* ------------------- */
+/* Custom properties   */
+/* ------------------- */
+
+:root {
+  --clr-dark: 0, 22, 50;
+  --clr-grey: 189, 193, 197;
+  --clr-grey-light: 211, 211, 211;
+  --clr-accent: 85, 172, 238;
+  --clr-white: 255, 255, 255;
+
+  --ff-global: "poppins", sans-serif;
+}
+
+/* ------------------- */
+/* Utility classes     */
+/* ------------------- */
+
+/* general */
+
+.center-wrapper {
+  padding-inline: 1.5em;
+  margin-inline: auto;
+  max-width: 100rem;
+}
+
+.text-align-center {
+  text-align: center;
+}
+
+/* grid classes */
+
+.grid {
+  display: grid;
+  gap: 1rem;
+}
+
+.grid-auto-flow {
+  display: grid;
+  grid-auto-flow: column;
+}
+
+.two-column-grid__expand-one {
+  display: grid;
+  grid-template-columns: 1fr auto;
+}
+.two-column-grid__expand-two {
+  display: grid;
+  grid-template-columns: auto 1fr;
+}
+
+.six-col-grid__expand-last {
+  display: grid;
+  grid-template-columns: repeat(5, 1fr) auto;
+}
+
+.three-row-grid__expand-two {
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+}
+
+.place-items-center {
+  display: grid;
+  place-items: center;
+}
+
+/* ------------------- */
+/* Components          */
+/* ------------------- */
+
+.btn {
+  background-color: rgba(var(--clr-accent));
+  padding: 10px;
+  border-radius: 2rem;
+  color: rgba(var(--clr-white));
+  font-weight: 600;
+}
+
+.tweet-btn {
+  margin-left: 10px;
+  width: 80%;
+  border: none;
+}
+
+.tweet-btn-sm {
+  font-size: 12px;
+  padding: 5px;
+  margin-left: 10px;
+  width: 80px;
+  border: none;
+  margin-inline: 10px;
+}
+
+.icon-ex-sm {
+  height: 15px;
+  aspect-ratio: 1;
```

Comment:
what is aspect-ratio solve here?

Reply:
it might be unnecessary, i just wanted to make sure its stayed as a perfect square - i guess i could've just set a width of 15px instead.